### PR TITLE
Log URLs downloaded by jsoc client

### PR DESCRIPTION
**Pull request by dstansby**
This might make it easier to diagnose things like https://github.com/sunpy/sunpy/issues/4555 by turning on the debug log.

### CODE REVIEW

File: sunpy/net/jsoc/jsoc.py
Comment (nabobalis) [inline sunpy/net/jsoc/jsoc.py:648]: Why do we have a print statement here?

**Review by nabobalis — APPROVED** [review]
Happy to merge this, but I wonder if we want to do this for all downloaders?

**Comment by dstansby** [comment]
Oddly this doesn't seem to be doing anything for me, so might be worth holding off until I can work out why it's not working.

Yes, I think rolling it out elsewhere would be good. Seems like a nice "good first issue"?

**Review by nabobalis — CHANGES_REQUESTED** [review]
Holding off.

File: sunpy/net/jsoc/jsoc.py
Comment (dstansby) [inline sunpy/net/jsoc/jsoc.py:650]: Well and truly baffled, this doesn't work (ie. a log message doesn't get printed), but if I put `print(log)` before this line it does...

Comment (nabobalis) [reply sunpy/net/jsoc/jsoc.py:650]: Yeah this looks like it should be fine.

Comment (ayshih) [reply sunpy/net/jsoc/jsoc.py:650]: > Well and truly baffled, this doesn't work (ie. a log message doesn't get printed), but if I put `print(log)` before this line it does...

Er, are you turning on debug output properly?
```python
log.setLevel("DEBUG")
```
I just tried this on my machine, and the debug log message shows up fine.

Comment (ayshih) [reply sunpy/net/jsoc/jsoc.py:650]:
```
DEBUG: Adding http://jsoc.stanford.edu/SUM14/D1333490357/S00000/hmi.m_45s.20200101_010130_TAI.2.magnetogram.fits to down
load queue. [sunpy.net.jsoc.jsoc]
```

Comment (dstansby) [reply sunpy/net/jsoc/jsoc.py:650]: Yeah, I'm setting the level properly. Maybe it's just something funny with my setup.

**Comment by Cadair** [comment]
This might be better done in parfive?

**Comment by dstansby** [comment]
> This might be better done in parfive?

That's a good idea, I shall close this in favour of some parfive logging

**Pull request closed without merging**
